app/views/groups/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1 class="text-center pb-1 bg-primary text-light">All Catagories </h1>
+
+<div class="container">
+  <%@groups.each do |group|%>
+  <div class="row justify-content-center">
+    <div class="card text-dark bg-light mb-3" style="width: 60%;">
+      <div class="row g-0 ">
+        <div class="col-md-4 w-75">
+          <p class="icon"><%= group.icon %></p>
+        </div>
+        <div class="col-md-8">
+          <div class="card-body">
+            <h5 class="card-title"><%=link_to group.name.capitalize , group_path(group), class:'text-secondary'%></h5>
+            <p class="card-text">Total Amount: $ <%= group.entities.sum(:amount) %></p>
+            <%= link_to 'View' , group_path(group), class:'btn btn-primary'%>
+            <p class="card-text text-color mt-2"><small class="text-muted">
+                created <%= time_ago_in_words(group.created_at) %> ago
+              </small></p>
+          </div>
+        </div>
```

Comment:
- [ ] There is a difference in the design, kindly follow the designs, especially the structure of the card.

## Yours

![cat](https://user-images.githubusercontent.com/53124548/204082012-af3c2ef1-f465-461d-ac32-0095662e0ece.png)

![design](https://user-images.githubusercontent.com/53124548/204082025-fd982203-a429-4aee-a280-4154a2ed208c.png)

## Orignal
![orignal](https://user-images.githubusercontent.com/53124548/204082078-fb7f850b-532c-4b8c-9596-c74095d6eecb.png)

Reply:
Transaction and cartegory styling fixed

![Screenshot (128)](https://user-images.githubusercontent.com/63410324/204090746-9a8dca3b-766d-4c47-99b3-72e25539e365.png)

![Screenshot (127)](https://user-images.githubusercontent.com/63410324/204090757-26ff006c-1f4e-402a-af68-04062275266b.png)